Libraries/Network/RCTNetworking.mm

```diff
@@ -19,6 +19,7 @@
 #import <React/RCTURLRequestHandler.h>
 #import <React/RCTUtils.h>
 
+#import "RCTBlobManager.h"
```

Comment:
We've setup Blob as an optional dependency on the native side, so this require isn't allowed. Can you do something like the `setContentHandler` in RCTWebSocketModule?